index.html

```diff
@@ -0,0 +1,781 @@
+                            <figcaption class="gray very-small-text ">
+                                <a href="#"></a>NASA/JPL-Caltech/University of Arizona</figcaption>
+                        </figure>
+                        <p><a href="#">How NASA Found the Ideal Hole on Mars to Land In</a>
+                            <br>
+                            <span class="gray very-small-text">July. 30</span>
+                        </p>
+                    </div>
+                </div>
+            </section>
+
+            <section class="footer-thumbnails">
+                <hr>
+                <h3>Editors' Picks</h3>
+                <div class="editors-picks">
+                    <div class="thumbnails">
+                        <figure>
+                            <img src="./images/thumbnail7.webp" alt="Woman about to get into a bus" width="250"
+                                height="180">
+                            <figcaption class="gray very-small-text ">
+                                <a href="#"></a>
+                            </figcaption>
+                        </figure>
+                        <p><a href="#">'A Woman Waiting to Get On Told Me That My Shoe Was Untied'</a>
+                            <br>
+                            <span class="gray very-small-text">Aug. 23</span>
+                        </p>
+                    </div>
+
+                    <div class="thumbnails">
+                        <figure>
+                            <img src="./images/thumbnail8.webp" alt="Woman wearing a red dress" width="250"
+                                height="180">
+                            <figcaption class="gray very-small-text ">
+                                <a href="#"></a>Image by The New York Times</figcaption>
+                        </figure>
+                        <p><a href="#">Born to the Internet, Grew Up Fast in a Pandemic</a>
+                            <br>
+                            <span class="gray very-small-text">Aug. 24</span>
+                        </p>
+                    </div>
+
+                    <div class="thumbnails">
+                        <figure>
+                            <img src="./images/thumbnail9.webp" alt="Couple on a couch" width="250" height="180">
+                            <figcaption class="gray very-small-text ">
+                                <a href="#"></a>Tom Sibley for The New York Times</figcaption>
+                        </figure>
+                        <p><a href="#">TA Couple Look for Their First Purchase in the 'Heart of Brooklyn.'
+                                But Where Is That?</a>
+                            <br>
+                            <span class="gray very-small-text">Aug. 23</span>
+                        </p>
+                    </div>
+                </div>
+            </section>
+        </div>
+    </main>
+    <footer class="footer">
```

Comment:
- [ ]  The main footer should also be centered. Please kindly fix it also.

![image](https://user-images.githubusercontent.com/60184961/91615153-9f444980-e98b-11ea-83f1-3465484c4d32.png)